crates/egui/src/containers/panel.rs

```diff
@@ -124,6 +126,13 @@ impl SidePanel {
         }
     }
 
+    /// Redirect shapes to another paint layer.
```

Comment:
It should be noted that this is only for the `show` functions, not `show_inside`